simplified-viewer-audiobook/src/main/java/org/nypl/simplified/viewer/audiobook/AudioBookPlayerActivity.kt

```diff
@@ -971,22 +971,18 @@ class AudioBookPlayerActivity :
       playerBookmark = playerBookmark
     )
 
-    val backgroundThread =
-      MoreExecutors.listeningDecorator(Executors.newFixedThreadPool(1))
-
-    backgroundThread.execute {
-      val wasDeleted = AudioBookHelpers.deleteBookmark(
-        bookmarkService = this.bookmarkService,
-        accountID = this.parameters.accountID,
-        bookmark = bookmark
-      )
-
-      if (wasDeleted) {
-        currentBookmarks.remove(bookmark)
+    this.bookmarkService.bookmarkDelete(
+      accountID = this.parameters.accountID,
+      bookmark = bookmark,
+      ignoreRemoteFailures = true
+    ).map { wasDeleted ->
```

Comment:
This `wasDeleted` variable doesn't need to exist now the method is a Unit and doesn't return a boolean, right?

Reply:
Oops!